ui/lib/kv/addon/components/kv-patch/editor/form.js

```diff
@@ -10,7 +10,7 @@ import { A } from '@ember/array';
 import { hasWhitespace, isNonString, WHITESPACE_WARNING, NON_STRING_WARNING } from 'vault/utils/validators';
 
 /**
- * @module KvPatchEditor::Form
+ * @module KvPatch::Editor::Form
```

Comment:
This is the parent component that renders `<KvPatch::Editor::Alerts>` and `<KvPatch::Editor::Row>`